taskt/Core/Automation/Commands/Text File Commands/WriteTextFileCommand.cs

```diff
@@ -41,7 +41,7 @@ public class WriteTextFileCommand : ScriptCommand
                             "in the file")]
         [SampleUsage("Select Append or Overwrite")]
         [Remarks("")]
-        public string v_Overwrite { get; set; }
+        public string v_Overwrite { get; set; } = "Append";
```

Comment:
lol no, I meant in the constructor. v_Overwrite = "Append"

Reply:
That doesn't make any difference, increase readability, or create a consistent UI experience in the application. 

The point of guidelines (of any type) is to *reduce* discussion in PRs so that we can focus on delivering a product. They are not so that we force developers to write things a certain way or in a certain place. I'm all for having a consistent UI experience which is why changes regarding tool tips and the such I've left completely up to you and Ali, that is not my domain. As a rule of thumb: a guideline is only as useful as much as it helps elucidate some piece of the code base. For this reason things like `_` in front of private fields are useful, because they give the developer context. This does not.

There is such a thing as too many guidelines and I think the line gets crossed when you start mandating how developers have to solve problems. There are plenty of places in the code base where we use this style of property defaulting, requiring that it be used in the constructor only in commands is going to be incredibly difficult to communicate to new developers and for existing developers it's going to just pollute their PRs and slow us down.

I don't think we need so many guidelines for commands, what we need is one person who has an idea of how the user experience is going to be and have that person make the changes to the UI components. The rest of the command code should just be subject to the regular .Net Core and .Net Framework guidelines like the rest of the code. That being said, unless you feel strongly about this I will not change it. Let me know.